src/Components/GraduatesList/FilterButton.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Dropdown, Input } from "semantic-ui-react";
+function FilterButton(props) {
+  const filterOptions = [
+    {
+      key: "Node",
+      text: "Nodo",
+      value: "Node",
+      label: { color: "red", empty: true, circular: true }
+    },
+    {
+      key: "Module",
+      text: "Modulo Cursado",
+      value: "Module",
+      label: { color: "blue", empty: true, circular: true }
+    },
+    {
+      key: "Level",
```

Comment:
Also here, not just level, key should be english-level

Reply:
I corrected that